cetmix_tower_server/models/cx_tower_server_template.py

```
@@ -0,0 +1,653 @@
+        # Prepare context for server creation
+        context = self.env.context.copy()
+
+        # SSH setting may be added after server creation.
+        context.update({"skip_ssh_settings_check": True})
+        # We need to remove default_server_template_id to avoid it being used
+        # in variable values.
+        context.pop("default_server_template_id", None)
+
+        # Create server
+        server = (
+            self.env["cx.tower.server"]  # pylint: disable=context-overridden # new need a new clean context
```

Pylint (4.0.3)

[warning] Unknown option value for 'disable', expected a valid pylint message and got 'context-overridden'

(W0012)